net-test/bin/txload.sh

```diff
@@ -0,0 +1,328 @@
+#!/usr/bin/env bash
+
+FEE_RATE=300
+MAX_CHAINING=5
+CONFIRMATIONS=1
+
+exit_error() {
+   printf "$1"
+   exit 1
+}
+
+for CMD in cut grep egrep sed blockstack-cli curl; do
```

Comment:
`sed` is not used (no longer?) in this script.  And with the latest, I still need to replace `sed`